app/services/sources/sources.ts

```diff
@@ -327,6 +328,13 @@ export class SourcesService extends StatefulService<ISourcesState> implements IS
       resolvedSettings.video_device_id = this.defaultHardwareService.state.defaultVideoDevice;
     }
 
+    // setup the list of supported games for the auto mode in the game_capture
+    if (type === 'game_capture') {
+      resolvedSettings.auto_capture_list_path = `${electron.remote.app.getPath(
+        'userData',
+      )}/game_capture_list.lst`;
+    }
```

Comment:
What happens if this file doesn't exist at this point?  Does the game capture gracefully handle this?  Will it find the list if it comes into existence later?  What if the file is only partially downloaded?  (We are not awaiting the download, not sure if that was intentional)

Reply:
game capture expect for a file to be readable at the moment settings set. it is better to set this option again after it downloaded.

Reply:
K.  We definitely need to be waiting on this download then, and it needs to be done much earlier in the initialization process.